content/pandas/concepts/built-in-functions/to-datetime/to-datetime.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: '.to_datetime()'
+Description: 'Reads inputs representation and returns it into data-time type'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Display'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science '
+---
+
+The **`to_datetime()`** function in the Pandas library of Python reads a string (or another) representation of a date and time into a datetime object.
+
+## Syntax
+
+This function returns a value in datetime format. Various input arguments can be used as shown and described below.
+
+```py
+pandas.to_datetime(arg, format=None, errors='raise', dayfirst=False, yearfirst=False, utc=None, box=True, infer_datetime_format=False, origin='unix', cache=True)
+```
+
+| Parameter name | Data type | Usage |
+| -------------- | --------- | ----- |
+| arg | int, float, str, datetime, list, tuple, 1-d array, Series, DateTime/dict-like | Converts given data into a datetime |
+| errors | 'ignore', 'raise', 'coerce' | returns specific exception depends in given parameter |
+| dayfirst | bool (default False) | if `True` is selected, the day will be given first |
+| yearfirst | bool (default True) | if `True` is selected, the year will be given first. If `dayfirst` and `yearfirst` is `True`, `yearfirst` is more important |
+| utc | bool (default None) | whenever `True` is selected, output is converted into UTC time zone |
+| format | str (default None) | helpfull in analyzing time, more info. in 'str' docs |
+| exact | bool (default True) | defines how format is used |
+| unit | str (default 'ns') | defines a unit, float or integer number |
+| infer_datetime_format | bool (default False) | whenever `True` is selected, datetime string format is based on `NaN` element
+| origin | scalar (default unix) | reference date is based on that |
+| cache | bool (default True) | is beeing used in datetime conversion, works only with at least 50 values |
+
+## Example
+
+First section of the following code shows how to import the needed library and how to create list of string. After that, it calls the `.to_datetime()` function and prints results onto the console.
+
+```py
+import pandas as pd
+my_list = ['2039/11/09']
+
+xyz = pd.to_datetime(my_list)
```

Comment:
```suggestion
xyz = pd.to_datetime(my_list, dayfirst=True)
```

Reply:
I thought it might be instructive to demonstrate how the use of one of the parameters determines the parsing of the object